src/Sentry.Unity.Editor/SentryAssetPostprocessor.cs

```diff
@@ -0,0 +1,52 @@
+using System.Linq;
+using System.Threading;
+using UnityEditor;
+using UnityEditor.PackageManager;
+using UnityEngine;
+
+namespace Sentry.Unity.Editor
+{
+    public class SentryAssetPostprocessor : AssetPostprocessor
+    {
+        private static void OnPostprocessAllAssets(string[] importedAssets, string[] deletedAssets, string[] movedAssets, string[] movedFromAssetPaths)
+        {
+            var imported = importedAssets.Any(path => path.StartsWith("Packages/"));
+            var deleted = deletedAssets.Any(path => path.StartsWith("Packages/"));
+
+            if (imported || deleted)
```

Comment:
Do we want to prompt also when asset deleted? I wonder the chance of false positive here

Reply:
Maybe not a prompt but maybe we should delete the assets we create too? Like link.xml and the options file?